packages/common/src/helpers.ts

```diff
@@ -1,3 +1,5 @@
+// eslint-disable-next-line eslint-comments/disable-enable-pair
+/* eslint-disable @typescript-eslint/no-unsafe-assignment */
```

Comment:
Do we need this eslint disablement code, anymore?

Reply:
Yes, for `require` statement, It's required. I have changed the disable rule for the file to the specific line.